### PR TITLE
Client side techdebt

### PR DESCRIPTION
**Pull request by wheresrhys**
@matthew-andrews

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,3 +5,4 @@ run:
 test:
 	nbt verify --skip-layout-checks
 	mocha
+	# ./node_modules/karma/bin/karma start
```
Comment (matthew-andrews) [inline Makefile]: ?

Comment (wheresrhys) [inline Makefile]: see commit message - there's a bug in fetch-mock (which I can't work on from here due to weird github ssh key issues) which I'll fix  from home. There were no karma tests before, so it's no worse than previously.

Comment (matthew-andrews) [inline Makefile]: fair enough

**Comment by matthew-andrews** [comment]
:+1:

**Pull request merged**